response_operations_ui/views/messages.py

```diff
@@ -132,6 +177,11 @@ def _refine(message):
     }
 
 
+def _get_survey_id(selected_survey):
+    survey_messages = survey_controllers.get_survey(selected_survey)
+    return survey_messages['survey']['id']
```

Comment:
Need to wrap this in a try/catch block.  If survey_messages['survey'] is somehow empty, then when it tries to get the ['id'] then it'll throw a typeError

Reply:
`return survey_controllers.get_survey(selected_survey)['survey']['id']`
??